specs/capella/beacon-chain.md

```diff
@@ -381,19 +280,55 @@ def process_block(state: BeaconState, block: BeaconBlock) -> None:
     process_sync_aggregate(state, block.body.sync_aggregate)
 ```
 
+#### New `get_expected_withdrawals`
+
+```python
+def get_expected_withdrawals(state: BeaconState) -> Sequence[Withdrawal]:
+    epoch = get_current_epoch(state)
+    withdrawal_index = state.next_withdrawal_index
+    index = ValidatorIndex((state.last_withdrawal_validator_index + 1) % len(state.validators))
+    ret: List[Withdrawal] = []
+    for probed in range(len(state.validators)):
+        val = state.validators[index]
+        balance = state.balances[index]
+        if is_fully_withdrawable_validator(val, balance, epoch):
+            withdrawal = Withdrawal(
+                index=withdrawal_index,
+                validator_index=index,
+                address=ExecutionAddress(val.withdrawal_credentials[12:]),
+                amount=balance,
+            )
+            ret.append(withdrawal)
+            withdrawal_index = WithdrawalIndex(withdrawal_index + 1)
```

Comment:
Does python require this syntax? A `withdrawal_index++` would look more readable, or use the increasing length from
```py
index=state.next_withdrawal_index + len(withdrawals)
```

Reply:
There's no increment operator in Python, the notation
```python
withdrawal_index += WithdrawalIndex(1)
```
would work, and in this case I believe even 
```python
withdrawal_index += 1
```
would, so I'll change to that and see if pyspec complains :)